core/src/main/java/feast/core/config/FeastProperties.java

```diff
@@ -90,8 +90,18 @@ SecurityProperties securityProperties() {
     /* The active Apache Beam runner name. This name references one instance of the Runner class */
     private String activeRunner;
 
-    /* If true only one IngestionJob would be created per source with all subscribed stores in it */
-    private Boolean consolidateJobsPerSource = false;
+    /* Job Coordinator related properties */
+    private CoordinatorProperties coordinator;
+
+    @Getter
+    @Setter
+    public static class CoordinatorProperties {
+      /* If true only one IngestionJob would be created per source with all subscribed stores in it */
+      private Boolean consolidateJobsPerSource = false;
+
+      /* Labels to filter by jobs managed by current instance */
```

Comment:
`Labels to filter by jobs managed by current instance` is a bit hard to undestand. What about `Labels to identify jobs managed by this job coordinator`?